example/spec/helpers/granuleUtils.js

```diff
@@ -144,11 +145,127 @@ async function setupTestGranuleForIngest(bucket, inputPayloadJson, granuleRegex,
   ])(baseInputPayload);
 }
 
-const deleteGranules = async (prefix, granules) => {
+async function generatePayloadGranule(passThrough, granule2Replicate, bucket, granuleRegex) {
+  const oldGranuleId = granule2Replicate.granuleId;
+  const newGranuleId = randomStringFromRegex(granuleRegex);
+
+  if (passThrough) {
+    // Setup for Ingest Passthrough
+    granule2Replicate.files = granule2Replicate.files.map(
+      (file) => ({ ...file, bucket: bucket })
+    );
+
+    granule2Replicate.createdAt = Date.now();
+  } else {
+    await createGranuleFiles(
```

Comment:
It was there as a side-effect in case of the `ingestLoad_queueGranSFCompleted` test scenario which would carry out the ingestion via official route with the existing Ingest workflow which would require the S3 data staging.

Reply:
Refactoring is in the latest code